commands/fork.go

```diff
@@ -102,8 +103,20 @@ func fork(cmd *Command, args *Args) {
 
 	args.NoForward()
 	if !flagForkNoRemote {
+
 		originURL := originRemote.URL.String()
 		url := forkProject.GitURL("", "", true)
+
+		// Check to see if the remote already exists and points to the fork.
+		currentRemote, err := localRepo.RemoteByName(newRemoteName)
+		if (err == nil) {
+
+			parsedURL, err := git.ParseURL(url)
+			if err == nil && parsedURL.String() == currentRemote.URL.String(){
```

Comment:
Rather than comparing string representations of two urls, let's rather construct Project instances using `remote.Project()` and use `project.SameAs(anotherProject)` to compare them. This is going to detect whether a git remote already exists for this project, but ignore the differences in protocol, which I think is desireable in this case.

For example, if a remote was going to be added for `https://github.com/foo/bar.git`, but one already exists for `git@github.com:foo/bar.git` or even `https://github.com/foo/bar` (no `.git` suffix), those differences are irrelevant and should be ignored.